src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -1,29 +1,95 @@
+import Button from '@common/components/Button';
 import LoadingSpinner from '@common/components/LoadingSpinner';
 import RelatedInformation from '@common/components/RelatedInformation';
-import { PublicationSummaryWithRelease } from '@common/services/publicationService';
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import { useMobileMedia } from '@common/hooks/useMedia';
+import useToggle from '@common/hooks/useToggle';
+import {
+  PublicationSummaryWithRelease,
+  PublicationSortOptions,
+} from '@common/services/publicationService';
 import { Paging } from '@common/services/types/pagination';
 import Page from '@frontend/components/Page';
 import Pagination from '@frontend/components/Pagination';
 import useRouterLoading from '@frontend/hooks/useRouterLoading';
+import styles from '@frontend/modules/find-statistics/FindStatisticsPage.module.scss';
 import PublicationSummary from '@frontend/modules/find-statistics/components/PublicationSummary';
+import SortControls from '@frontend/modules/find-statistics/components/SortControls';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+import classNames from 'classnames';
 import { NextPage } from 'next';
-import React from 'react';
+import { useRouter } from 'next/router';
+import React, { useEffect, useState } from 'react';
 
 interface Props {
   paging: Paging;
   publications: PublicationSummaryWithRelease[];
+  sortBy?: PublicationSortOptions;
 }
 
-const FindStatisticsPageNew: NextPage<Props> = ({ paging, publications }) => {
+const FindStatisticsPageNew: NextPage<Props> = ({
+  paging,
+  publications,
+  sortBy,
+}) => {
   const { page, totalPages, totalResults } = paging;
-  const isLoading = useRouterLoading();
+  const router = useRouter();
+  const isRouterLoading = useRouterLoading();
+  const { isMedia: isMobileMedia } = useMobileMedia();
+  const [currentPage, setCurrentPage] = useState<number>(page);
+  const [currentPublications, setCurrentPublications] = useState<
+    PublicationSummaryWithRelease[]
+  >(publications);
+  const [currentSortBy, setCurrentSortBy] = useState<
+    PublicationSortOptions | undefined
+  >(sortBy ?? undefined);
+  const [isLoadingPublications, toggleIsLoadingPublications] = useToggle(false);
+
+  useEffect(() => {
+    setCurrentPage(page);
+    setCurrentPublications(publications);
+  }, [page, publications]);
```

Comment:
How come we're doing everything in state again?

Noticed your comment that there are 'usability problems as the page reloads while you're interacting with the sort controls' . What actually happens?

I stripped out all the state and changed the `router.push` back to be non-shallow and everything seems to work fine. Focus is not lost and I can cycle through the sort options easily enough. Is there a screen reader issue here?

Reply:
I'm not sure what I was doing before to make it not work properly! I've taken it back out again and all works fine.